### PR TITLE
Run tests for PR by using travis

### PR DESCRIPTION
**Pull request by aaltat**

### CODE REVIEW

**Comment by pekkaklarck** [comment]
Thanks for the PR! Getting a proper CI system is very important. I won't have time to look at this closely in the near future, but please keep experimenting and let me know when the setup is getting ready. We can discuss here or on Slack about the requirements a CI system should have in our usage.

**Comment by windelbouwman** [comment]
@aaltat : nice job! I was wondering why this was not in place yet!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - python: "3.4"
+      env:
+        - INTERPRETER=python3
+before_script:
+    $INTERPRETER -m pip install -r atest/requirements.txt
+script:
+  - echo $INTERPRETER
+  - $INTERPRETER --version
+  - $INTERPRETER utest/run.py
+  - $INTERPRETER atest/run.py $INTERPRETER --exclude no-ci atest/robot
```
Comment (windelbouwman) [inline .travis.yml]: Idea: comment this out for now. First get unittests running on jenkins, then later add acceptance tests. Also, please add a badge (like `[Build Status](https://travis-ci.org/RustPython/RustPython)`)

**Comment by hugovk** [comment]
See also https://github.com/robotframework/robotframework/pull/2676.

**Comment by aaltat** [comment]
There was another PR, which allows also runs in Windows. Will close this one.

**Pull request closed without merging**